Add warning for users of Google reCAPTCHA version 3 with low score (#1597)
magento/merchdocs: src/configuration/security/google-recaptcha-admin.md

{: .bs-callout-info}
The reCAPTCHA type you choose must match the type that is associated with the API key from your Google reCAPTCHA account.

{:.bs-callout-warning}

> When using version 3, a genuine user with low score will be unable to proceed; when using version 2, a genuine user with a low score will receive an challenge. Consider carefully if genuine users with a low score should be given an opportunity to solve a challenge (version 2) or completely blocked (version 3).

Reviewer: ```suggestion
When using reCAPTCHA version 3, a genuine user with low score cannot proceed. For version 2, a genuine user with a low score receives a challenge. Consider carefully if genuine users with a low score should have an opportunity to solve a challenge (version 2) or be completely blocked (version 3).
```
